lib/lotus/model.rb

```diff
@@ -25,13 +28,13 @@ class NonPersistedEntityError < ::StandardError
     # @since 0.2.0
     #
     # @see Lotus::Configuration#mapping
-    class InvalidMappingError < ::StandardError
+    class InvalidMappingError < Error
```

Comment:
:clap: nice work
while you are on it, can we add test for every exception class?